pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -29,7 +29,6 @@ import (
 
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
 	ktypes "k8s.io/apimachinery/pkg/types"
```

Comment:
Would it be better to use `"k8s.io/apimachinery/pkg/types"` instead of `ktypes "k8s.io/apimachinery/pkg/types"`?

Reply:
Yup. Seems other places are all using "k8s.io/apimachinery/pkg/types" directly, and only scheduler code are using the "ktypes" alias:

```
pkg/scheduler/internal/queue/scheduling_queue.go
34:	ktypes "k8s.io/apimachinery/pkg/types"

pkg/scheduler/internal/queue/scheduling_queue_test.go
31:	ktypes "k8s.io/apimachinery/pkg/types"
```